ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -16,7 +16,6 @@
 package io.confluent.ksql.util;
 
 import static io.confluent.ksql.configdef.ConfigValidators.zeroOrPositive;
-import static org.apache.kafka.streams.StreamsConfig.InternalConfig.ENABLE_KSTREAMS_OUTER_JOIN_SPURIOUS_RESULTS_FIX;
```

Comment:
qq: Since this is introduced in KS only for ksqlDB, could we remove the config in KS as well? If yes I will file a quick PR.

Reply:
Not sure. @mjsax should we remove the config from KS now that the fix is automatically enabled when using ofTimeAndGracePeriod()?

Reply:
Don't have a strong opinion. But gives that we piggy-back opt-in/opt-out into the new semantics via API/language, it seems ok to remove both.